tests/backend/test_cli.py

```diff
@@ -0,0 +1,42 @@
+import os
+
+from click.testing import CliRunner
+from writer.command_line import main
+
+
+def test_version():
+   runner = CliRunner()
+   with runner.isolated_filesystem():
+      result = runner.invoke(main, ['-v'])
+      assert result.exit_code == 0
+      assert 'version' in result.output
+
+def test_create_default():
+    runner = CliRunner()
+    with runner.isolated_filesystem():
+        result = runner.invoke(main, ['create', './my_app'])
+        print(result.output)
```

Comment:
Is this one necessary ?

Reply:
thats depends, pytest is capturing prints and display them only when test fails and when it fails this print is really useful.
When test passes then it's useless but also not displayed... at least when you use the default reporter.